reporter/src/main/kotlin/model/EvaluatedScanResult.kt

```diff
@@ -36,5 +38,6 @@ data class EvaluatedScanResult(
     val endTime: Instant,
     val fileCount: Int,
     val packageVerificationCode: String,
```

Comment:
Just noticed, was there a reason not to add `@JsonIgnore` to the `packageVerificationCode`?

Reply:
I assumed that it would not be empty.

Reply:
Added it.